### PR TITLE
fix(NetEase Cloud Music): fix and improve usage

### PR DESCRIPTION
**Pull request by Kotoki1337**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image

</details>

### CODE REVIEW

File: websites/#/网易云音乐/metadata.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
 		"vi_VN": "Netease Cloud Music là nền tảng phát nhạc trực tuyến Freemium được sở hữu và phát triển bởi Tập đoàn NetEase."
 	},
 	"url": "music.163.com",
-	"version": "1.0.12",
+	"version": "1.0.13",
```
Comment (EncryptedDev) [inline websites/#/网易云音乐/metadata.json:16]: Changing the clientId is a breaking change

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Comment by theusaf** [comment]
Shouldn't changes made to Stake's metadata be undone first?

**Comment by Kotoki1337** [comment]
it's literally `yarn format`, should i revert it (the Stake one)?

**Comment by Bas950** [comment]
> it's literally `yarn format`, should i revert it (the Stake one)?

Either, bump it's version, or revert that change please.

**Review by Bas950 — APPROVED** [review]

**Pull request merged**